priv/repo/migrations/20200305174544_add_slug_value_for_existing_companies.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Companies.Repo.Migrations.AddSlugValueForExistingCompanies do
```

Comment:
I don't know how Heroku works with elixir, but when I tried to do this in the past, while it worked for my local environment, it failed miserably for production, since the migration scripts did not have any access to the actual app modules when they run, and therefore they failed. @doomspork / @maartenvanvliet ?

Reply:
Yeah, locally it worked but I don't know about production env. An alternative to this would be run this slug generation one time in heroku.

Reply:
Yes, restricting the migrations to just ddl might be preferable. There was an interesting post by AppSignal on doing this in Elixir (https://blog.appsignal.com/2020/02/25/migrating-production-data-in-elixir.html)

That might be overkill in this case and a one time command would also work.